db-drda/src/main/java/com/splicemachine/db/impl/drda/DRDAConnThread.java

```diff
@@ -1188,6 +1200,9 @@ private void processCommands() throws DRDAProtocolException
                     server.consoleExceptionPrint(sqle);
                     SanityManager.THROWASSERT("Unexpected exception after " +
                             "codePoint "+cpStr, sqle);
+                } catch (Exception e) {
+                    server.consoleExceptionPrint(e);
```

Comment:
Use `LOG`. Ideally we'll have just one logging system and it won't be the Derby one.

Reply:
Yes, that's definitely something we should do, I didn't want to introduce that here because I thought it is out of scope. Will create a JIRA for this at least.

For now, I would prefer to keep it like this, because the error will appear anyway (although in a different log file).